src/app/icd/ICDManager.h

```diff
@@ -108,6 +113,11 @@ class ICDManager : public ICDListener
     ICDStateObserver * mStateObserver              = nullptr;
     bool mTransitionToIdleCalled                   = false;
     Crypto::SymmetricKeystore * mSymmetricKeystore = nullptr;
+
+#ifdef CONFIG_BUILD_FOR_HOST_UNIT_TEST
```

Comment:
This should be the default not just for unit test. The feature map logic should be runtime given the complexity of the feature, to avoid mistakes with static configs in ZAP.

Reply:
I'm not sure what is the take away is... Is the suggestion not to use the configured value in ZAP for the featuremap?

Reply:
resolving this for now. This can be updated in a follow up if necessary.

Reply:
Yes, have the server infrastructure maintain the featuremap, so it doesn't need to be compile-time-conditionally updated.